client/components/domains/register-domain-step/index.jsx

```diff
@@ -57,20 +59,26 @@ function processSearchQueue() {
 					continue outerLoop;
 				}
 			}
-			reportSearch( queue[ i ] );
+			reportSearchStats( queue[ i ] );
 		}
 }
 
-function reportSearch( { query, section } ) {
-	analytics.recordEvent( 'searchFormSubmit', query, section );
+function reportSearchStats( { query, section, timestamp } ) {
+	let timeDiffFromLastSearchInSeconds = 0;
+	if ( lastSearchTimestamp ) {
+		timeDiffFromLastSearchInSeconds = Math.floor( ( timestamp - lastSearchTimestamp ) / 1000 );
+	}
+	lastSearchTimestamp = timestamp;
+	searchCount += searchCount;
```

Comment:
As mentioned in Slack, my code-review senses are tingling on this line... I'm not _quite_ sure why :wink: